app/javascript/components/region-form/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {Component, useState} from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './region-form.schema';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import {API} from '../../http_api';
+import handleFailure from "../../helpers/handle-failure";
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+
+class RegionForm extends Component {
```

Comment:
Instead of a `class` component could you make a `function` component? Then you can eliminate the `constructor` function and have just `useState` below.